### PR TITLE
[srvls] [SRVKS-693] Route docs for serverless

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-openshift-routes.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+spec:
+  template:
+    metadata:
+      annotations:
+        openshift.io/disableRoute: true
+        # Check this with Kenjiro
```
Comment (nak3) [inline modules/serverless-openshift-routes.adoc:29]: This annotations needs to be configured like:

```
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: <service_name>
  annotations:
    openshift.io/disableRoute: true
spec:
  template:
    ... 
```

File: modules/serverless-openshift-routes.adoc [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE----
+  wildcardPolicy: None
+----
+<1> The timeout value for the {product-title} route. You should set the same value as the `max-revision-timeout-seconds` setting (`600s` by default).
+<2> The name of {product-title} route.
+<3> The namespace for the {product-title} route. This must be `knative-serving-ingress`.
+<4> The hostname for external access. You can set this to `<service_name>-<service_namespace>.<domain>` or your custom domain if you have configured one.
```
Comment (nak3) [inline modules/serverless-openshift-routes.adoc:97]:
```suggestion
<4> The hostname for external access. You can set this to `<service_name>-<service_namespace>.<domain>`.
```

The custom domain needs a combination of [DomainMapping](https://docs.openshift.com/container-platform/4.6/serverless/networking/serverless-domain-mapping.html).
I will send the PR once this PR was merged.

File: modules/serverless-openshift-routes.adoc [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  host: <service_host> <4>
+  port:
+    targetPort: http2
+  To:
+    kind: Service
+    name: kourier
+    weight: 100
+  Tls:
+    insecureEdgeTerminationPolicy: Allow
+    termination: edge
```
Comment (nak3) [inline modules/serverless-openshift-routes.adoc:79]:
```suggestion
    termination: edge <5>
    
    ...
    
<5> Type of TLS termination. Only `edge` termination is supported.
```

This is asked very often. We can support edge termination only.

**Review by nak3 — COMMENTED** [review]
Thank you! I left a few comments.

**Comment by nak3** [comment]
/lgtm

Thank you!

File: modules/serverless-openshift-routes.adoc [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 600s <1>
+  name: <route_name> <2>
+  namespace: knative-serving-ingress <3>
+spec:
+  host: <service_host> <4>
+  port:
+    targetPort: http2
+  To:
```
Comment (maschmid) [inline modules/serverless-openshift-routes.adoc:71]: lowercase "to"

File: modules/serverless-openshift-routes.adoc [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    kind: Service
+    name: kourier
+    weight: 100
+  Tls:
```
Comment (maschmid) [inline modules/serverless-openshift-routes.adoc:75]: lowercase "tls"

**Comment by maschmid** [comment]
QE approved

File: modules/serverless-openshift-routes.adoc [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE----
+  wildcardPolicy: None
+----
+<1> The timeout value for the {product-title} route. You should set the same value as the `max-revision-timeout-seconds` setting (`600s` by default).
```
Comment (codyhoag) [inline modules/serverless-openshift-routes.adoc:92]: Is there a way to remove `should` from this? Is `must` too strong in this case?

>! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

File: modules/serverless-openshift-routes.adoc [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+<2> The name of {product-title} route.
```
Comment (codyhoag) [inline modules/serverless-openshift-routes.adoc:93]: s/of {product-title} route/of the {product-title} route

File: modules/serverless-openshift-routes.adoc [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+<3> The namespace for the {product-title} route. This must be `knative-serving-ingress`.
+<4> The hostname for external access. You can set this to `<service_name>-<service_namespace>.<domain>`.
+<5> The certificates you want to use. Currently only `edge` termination is supported.
```
Comment (codyhoag) [inline modules/serverless-openshift-routes.adoc:96]: s/Currently only/Currently, only

**Review by codyhoag — COMMENTED** [review]
Just a few nits; looks good otherwise!

**Pull request merged**